docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+  - Common
+    - Some memory leak bugs have been fixed.
+
+
+### UI framework
+
+#### New and changed features
+
+- EFL
+  - Version 1.24 has been upgraded to version 1.25.
+  - Support for gengrid has been added for wearable profile.
+  - Support for evas vector graphic has been added.
+  - Support for relative container has been added.
+
+
+### Multimedia framework
+
+#### New and changed features
+
+- Camera
+  - Support for platform privilege has been added.
+- MediaStreamer
+  - Native WebRTC API set has been added.
+    - MEDIA_STREAMER_PARAM_WEBRTC_PEER_TYPE
+    - MEDIA_STREAMER_PARAM_WEBRTC_STUN_SERVER
+    - MEDIA_STREAMER_PARAM_WEBRTC_REMOTE_SESSION_DESCRIPTION
+    - MEDIA_STREAMER_PARAM_WEBRTC_ADD_ICE_CANDIDATE
+    - MEDIA_STREAMER_PARAM_WEBRTC_RTP_TRANSCEIVER_DIRECTION_FOR_AUDIO
+    - MEDIA_STREAMER_PARAM_WEBRTC_RTP_TRANSCEIVER_DIRECTION_FOR_VIDEO
+    - MEDIA_STREAMER_PARAM_MAX_LATENESS
+    - MEDIA_STREAMER_PARAM_RTP_LATENCY
+    - media_streamer_node_decoded_ready_cb
+    - media_streamer_node_set_decoded_ready_cb
+    - media_streamer_node_unset_decoded_ready_cb
+    - media_streamer_webrtc_node_set_message_cb
+    - media_streamer_webrtc_node_unset_message_cb
+- Image Util
+  - Support for WebP has been added.
+  - Support for encoding for GIF and WebP animation has been added.
+  - Support for ARGB and GBRA for PNG encoding has been added.
+- Metadata Editor
+  - Support for WAV, FLAC, and OGG has been added.
+- Media Controller
+  - Result codes for command have been added.
+  - Three Playback states have been added. (Connection, Buffering, and Error)
+- Media Vision
+  - New enumerations for human pose landmark and body part have been added.
+  - API set to detect pose landmark, i.e., human body pose, has been added.
+- Recorder
+  - API set to decide video frame encoding has been added.
+  - Support for video scaling has been added.
+    - It is only for single video stream supported device.
+
+
+### Network and Connectivity
+
+#### New and changed features
+
+- Connection API
+  - Following API set to require new partner level privilege has been changed.
+    - New privilege: http://tizen.org/privilege/network.route
+    - Privilege level: partner
+    - Affected API set
+      - connection_add_route
+      - connection_remove_route
+      - connection_add_route_ipv6
+      - connection_remove_route_ipv6
+      - connection_add_route_entry
+      - connection_remove_route_entry
+- Bluetooth mesh network
+  - Bluetooth framework has been extended to support Bluetooth mesh network standard.
+    - Platform API for Bluetooth mesh network features has been added.
+    - Open source bluez module has been upgraded to support Bluetooth mesh network.
+    - Tizen Bluetooth framework has been extended to control bluez’s mesh daemon.
+  - Tizen devices will be able to do Bluetooth mesh network provisioning and control devices like smart blubs, which support Bluetooth mesh network standard.
```

Comment:
Please check for technical correctness, then only incorporate the comment. 
```suggestion
  - Tizen devices have been provisioned for Bluetooth mesh network and control devices like smart blubs, which support Bluetooth mesh network standards.

Reply:
@mijin85cho yes seems fine and crisp too.

Reply:
`Bluetooth mesh network provisioning and controlling devices like smart bulbs, which support Bluetooth mesh network standards has been added.`

looks good.

Reply:
@semun-lee  but without code styling. 
Thanks